index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <main>
+        <header>
+            <a class="logo-placeholder" href="#">John Doe</a>
+            <nav>
+                <button type="button"><img src="img/Union.png" alt="image of an icon, conformed by 3 lines"></button>
+            </nav>
+        </header>
+        <section>
+            <h1>Hey There. I’m Alex</h1>
+            <h3>I’am a Software Developer</h3>
+            <h5>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</h5>
```

Comment:
- [ ] [OPTIONAL] Kindly don't skip levels of `<h>` tags and a `<p>` tag would be more appropriate for a description.